Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook

```diff
@@ -3516,7 +3474,2305 @@
+                                "label": "Free Memory including File System Cache (MB)"
+                              },
+                              {
+                                "seriesName": "SESSIONS_d",
+                                "label": "Sessions"
+                              },
+                              {
+                                "seriesName": "USERS_d",
+                                "label": "User Logons"
+                              }
+                            ]
+                          }
+                        },
+                        "customWidth": "50",
+                        "name": "User Logons and Sessions",
+                        "styleSettings": {
+                          "maxWidth": "50",
+                          "showBorder": true
+                        }
+                      },
+                      {
+                        "type": 3,
+                        "content": {
+                          "version": "KqlItem/1.0",
+                          "query": "let appServerList = dynamic([{CpuMemApplicationServer}]);\r\nlet baseQuery = SapNetweaver_SMON_CL\r\n| where timestamp_t between (todatetime({CpuMemTimeRange:start}) .. todatetime({CpuMemTimeRange:end}))\r\n| where SID_s == '{CpuMemSID}'\r\n| project UPDQ_d, ENQQ_d, DIAQ_d, QINLENGTH_d, QOUTLENGTH_d, SID_s, hostname_s, instanceNr_s, SERVER_s, timestamp_t\r\n| extend instanceNr_s = iff(strlen(instanceNr_s) == 1, strcat(\"0\", instanceNr_s), instanceNr_s)\r\n| extend appServer = strcat(hostname_s, \"_\", SID_s, \"_\", instanceNr_s)\r\n| where array_length(appServerList) == 0 or strcat(hostname_s, \"_\", instanceNr_s) in (appServerList)\r\n| project UPDQ_d, ENQQ_d, DIAQ_d, QINLENGTH_d, QOUTLENGTH_d, timestamp_t;\r\nlet timespanCount = baseQuery \r\n| summarize RecordCount=count()\r\n| extend Interval = iff(RecordCount <= 10, timespan(1m), timespan(1m))\r\n| extend Interval = iff(RecordCount > 10 and RecordCount < 50, timespan(5m), Interval)\r\n| extend Interval = iff(RecordCount >= 50 and RecordCount < 100, timespan(10m), Interval)\r\n| extend Interval = iff(RecordCount >= 100 and RecordCount < 150, timespan(15m), Interval)\r\n| extend Interval = iff(RecordCount >= 150, timespan(30m), Interval)\r\n| project Interval;\r\nlet timespanInterval = toscalar(timespanCount);\r\nbaseQuery\r\n| summarize Count = count() by bin(timestamp_t, timespanInterval),UPDQ_d, ENQQ_d, DIAQ_d, QINLENGTH_d, QOUTLENGTH_d\r\n| project-away Count\r\n",
+                          "size": 0,
+                          "title": "Queue Lengths Performance",
+                          "queryType": 0,
+                          "resourceType": "microsoft.operationalinsights/workspaces",
+                          "visualization": "linechart",
+                          "chartSettings": {
+                            "xAxis": "timestamp_t",
+                            "seriesLabelSettings": [
```

Comment:
more series label cleanup please, remove all series that are not displayed in this graph